srtcore/handshake.h

```diff
@@ -311,6 +311,8 @@ class CHandShake
     // Applicable only when m_iVersion == HS_VERSION_SRT1
     int32_t flags() { return m_iType; }
 
+    bool v5() { return m_iVersion > 4; }
```

Comment:
`v5` should check for v5.
```suggestion
    bool v5() { return m_iVersion == 5; }
```

Reply:
No. The intention is that v5() return the handshake version that is at least 5. I don't know if we ever introduce version 6 handshake, but even if, these rules for extensions will be still in force. I could at best agree for `m_iVersion != 4`. This is because version 4 is the very first one.

Reply:
Just the function name implies it checks for v5.

Reply:
Then maybe change the name - `atleast_v5`? `v5plus`?

Reply:
Maybe `v5orHigher()`?